devops/scripts/load_env.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -o errexit
+# set -o errexit
```

Comment:
will this be commented out in the published PR?

Reply:
Ah no - that should be removed since we `source` the script rather than calling it. Thanks!